src/App/App.tsx

```diff
@@ -6,26 +6,38 @@ import Header from '../Header/Header';
 import GenresList from '../GenresList/GenresList';
 import PlaylistContainer from '../PlaylistContainer/PlaylistContainer';
 
-class App extends Component<{}> {
+interface IProps {}
+interface IState {
+	genres: [],
+	playlists: [],
+	selectedGenre: string
+}
+
+class App extends Component<IProps, IState> {
```

Comment:
Keeping the data that is being used under control is definitely the key with Typescript. Really appreciate this.